plugins/modules/swim_intent.py

```diff
@@ -81,19 +81,20 @@
                 elements: dict
                 suboptions:
                   application_type:
-                    description: Swim Import Via Url's applicationType.
+                    description: Optional parameter indicating the type of application with permitted values(WLC, LINUX, FILREWALL, WINDOWS,
+                      LOADBALANCER, THIRDPARTY etc) applicable only for third party image types.
                     type: str
                   image_family:
-                    description: Swim Import Via Url's imageFamily.
+                    description: The name of image family applicable only in case of third party images upload (Optional).
```

Comment:
Represents the name of the image family and is applicable only when uploading third-party images.

Reply:
added.